common/src/View/InputEvent.cpp

```diff
@@ -209,30 +209,30 @@ namespace TrenchBroom {
         InputEventRecorder::InputEventRecorder() :
         m_dragging(false),
         m_anyMouseButtonDown(false),
-        m_lastClickX(0),
-        m_lastClickY(0),
+        m_lastClickX(0.0),
+        m_lastClickY(0.0),
         m_lastClickTime(std::chrono::high_resolution_clock::now()),
         m_nextMouseUpIsRMB(false),
         m_nextMouseUpIsDblClick(false) {}
 
 
-        void InputEventRecorder::recordEvent(const QKeyEvent* wxEvent) {
-            m_queue.enqueueEvent(std::make_unique<KeyEvent>(getEventType(wxEvent)));
+        void InputEventRecorder::recordEvent(const QKeyEvent& qEvent) {
+            m_queue.enqueueEvent(std::make_unique<KeyEvent>(getEventType(qEvent)));
         }
 
-        void InputEventRecorder::recordEvent(const QMouseEvent* wxEvent) {
-                  auto type = getEventType(wxEvent);
-                  auto button = getButton(wxEvent);
-            const auto posX = wxEvent->x();
-            const auto posY = wxEvent->y();
+        void InputEventRecorder::recordEvent(const QMouseEvent& qEvent) {
+                  auto type = getEventType(qEvent);
+                  auto button = getButton(qEvent);
+            const auto posX = qEvent.localPos().x();
```

Comment:
I'd rather no `auto` on primitive types - I can't tell what this is from Github

Reply:
Also could use a comment that posX/Y are:
- units of Points
- coordinate system is relative to the widget (top-left corner?)

Reply:
I've added that info to the `MouseEvent` type.